app/javascript/components/NamedAvatar.js

```diff
@@ -16,17 +19,78 @@ const styles = {
   name: {
     fontWeight: 600,
   },
-  subtitle: {},
+  remove: {
+    marginLeft: '20px',
+    cursor: 'pointer',
+  },
+  dialogActionsLink: {
+    padding: '15px',
+    cursor: 'pointer',
+    fontSize: '16px',
+  },
 }
 
-const NamedAvatar = props => (
+const dialogActions = (togglePopover, onRemove) => [
+  <a
+    style={styles.dialogActionsLink}
+    onClick={() => {
+      togglePopover('removeUserFromEvent', false)
+    }}
+  >
+    Cancel
+  </a>,
+  <a
+    style={styles.dialogActionsLink}
+    onClick={() => {
+      onRemove()
+      togglePopover('removeUserFromEvent', false)
+    }}
+  >
+    Delete
+  </a>,
+]
+
+const NamedAvatar = ({ name, subtitle, showRemove, onRemove, togglePopover, popover }) => (
   <div style={styles.container}>
-    <Avatar {...props} />
+    <Avatar name={name} subtitle={subtitle} />
     <div style={styles.details}>
-      <span style={styles.name}>{props.name}</span>
-      <span style={styles.subtitle}>{props.subtitle || '\u00a0'}</span>
+      <span style={styles.name}>{name}</span>
+      <span style={styles.subtitle}>{subtitle || '\u00a0'}</span>
     </div>
+    {showRemove && (
+      <a
+        style={styles.remove}
+        onClick={() => {
+          togglePopover('removeUserFromEvent', true)
+        }}
+      >
+        Remove
+      </a>
+    )}
+    {!!popover ? (
```

Comment:
nit: this is just fine for people who know about JS trickery 😈 but since we're already using Ramda, how would you feel about inverting this with a more explicit function?

```js
R.isNil(popover) ? null : <Dialog />.
```

It's not the best handling of truthy, but at least it handles `null` _and_ `undefined`

Reply:
(i also realize i've written `!!` a ton... so it would take a while to replace it everywhere) do whatever you like!

Reply:
I would prefer the `R.isNil` check. We have done a good job of not using `!!` and truthy/falsy stuff in this project so far.